modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -97,8 +81,65 @@ import canvas from '@workday/canvas-kit-react/tokens';
 import {Button} from '@workday/canvas-kit-react/button';
```

Comment:
It's a little confusing how the slash imports example also includes a change in the import statement due to the removal of the default export from `Button`. It may be clearer to include a before/after example where the **only** change is due to the slash imports system.